sinatra-contrib/spec/spec_helper.rb

```diff
@@ -1,7 +1,82 @@
 ENV['RACK_ENV'] = 'test'
 require 'sinatra/contrib'
 
+# This file was generated by the `rspec --init` command. Conventionally, all
```

Comment:
This configuration has been _copied_ from `rack-protection` so they are the same